configs/dab_detr/dab_detr_r50_8xb2-50e_coco.py

```diff
@@ -0,0 +1,148 @@
+]
+model = dict(
+    type='DABDETR',
+    data_preprocessor=dict(
+        type='DetDataPreprocessor',
+        mean=[123.675, 116.28, 103.53],
+        std=[58.395, 57.12, 57.375],
+        bgr_to_rgb=True,
+        pad_size_divisor=1),
+    backbone=dict(
+        type='ResNet',
+        depth=50,
+        num_stages=4,
+        out_indices=(3, ),
+        frozen_stages=1,
+        norm_cfg=dict(type='FrozenBN2d', requires_grad=False),
+        norm_eval=True,
+        style='pytorch',
+        init_cfg=dict(type='Pretrained', checkpoint='torchvision://resnet50')),
+    encoder_cfg=dict(
+        num_layers=6,
+        layer_cfg=dict(
+            self_attn_cfg=dict(embed_dims=256, num_heads=8, dropout=0.),
+            ffn_cfg=dict(
+                embed_dims=256,
+                feedforward_channels=2048,
+                num_fcs=2,
+                ffn_drop=0.,
+                act_cfg=dict(type='PReLU')))),
+    decoder_cfg=dict(  # TODO: Need to refactor
+        num_layers=6,
+        query_dim=4,
+        query_scale_type='cond_elewise',
+        modulate_hw_attn=True,
+        layer_cfg=dict(
+            self_attn_cfg=dict(
+                embed_dims=256, num_heads=8, attn_dropout=0., proj_drop=0.),
+            cross_attn_cfg=dict(
+                embed_dims=256,
+                num_heads=8,
+                attn_dropout=0.,
+                proj_drop=0.,
+                cross_attn=True),  # TODO: add a param to determine msa or mca
+            ffn_cfg=dict(
+                embed_dims=256,
+                feedforward_channels=2048,
+                num_fcs=2,
+                ffn_drop=0.,
+                act_cfg=dict(type='PReLU'))),
+        return_intermediate=True),
+    positional_encoding_cfg=dict(
+        num_feats=128, temperatureH=20, temperatureW=20,
+        normalize=True),  # TODO: SinePositionalEncodingHW
+    bbox_head=dict(  # TODO: need to refactor
+        type='DABDETRHead',
+        num_classes=80,
+        embed_dims=256,
+        loss_cls=dict(
+            type='FocalLoss',
+            use_sigmoid=True,
+            gamma=2.0,
+            alpha=0.25,
+            loss_weight=1.0),  # TODO: weight of cls loss and cls cost
+        loss_bbox=dict(type='L1Loss', loss_weight=5.0),
+        loss_iou=dict(type='GIoULoss', loss_weight=2.0)),
+    # training and testing settings
+    train_cfg=dict(
+        assigner=dict(
+            type='HungarianAssigner',
+            match_costs=[
+                dict(type='FocalLossCost', weight=2., eps=1e-8),
+                dict(type='BBoxL1Cost', weight=5.0, box_format='xywh'),
+                dict(type='IoUCost', iou_mode='giou', weight=2.0)
+            ])),
+    test_cfg=dict(max_per_img=300))
+
+# train_pipeline, NOTE the img_scale and the Pad's size_divisor is different
+# from the default setting in mmdet.
+train_pipeline = [
+    dict(
+        type='LoadImageFromFile',
+        file_client_args={{_base_.file_client_args}}),
```

Comment:
It can represent as `_base_.file_client_args` in the latest mmengine, which is more clear